ui/src/components/canvas/FlowCanvas.tsx

```diff
@@ -68,8 +78,13 @@ const FlowCanvas = () => {
     clearSelectedChildNode,
     clearFlow,
     importFlowFromJson,
+    updateLayoutOrientation,
   } = useAppActions()
 
+  useEffect(() => {
+    reactFlowInstance.fitView()
+  })
```

Comment:
This will fit the view after every flow render. We should probably restrict this to `fitView` only after layouting changes, so you might want to add a dependency array containing layout settings (and `reactFlowInstance`) to the `useEffect` hook.